src/app/project/project-groups/group-contact/group-contact.component.ts

```diff
@@ -91,8 +91,9 @@ export class GroupContactComponent implements OnInit, OnDestroy {
       .subscribe((res: any) => {
         // Incoming users
         if (res && res.users && res.users.length > 0) {
```

Comment:
should probably change this condition as well to res.users[0].total_items > 0

Reply:
Yep, missed the if. Updating now